packages/nuxt/README.md

```diff
@@ -140,12 +140,24 @@ Add an import flag to the node options, so the file loads before any other impor
 }
 ```
 
-### 5. Vite Setup
-
-todo: add vite setup
-
 ---
 
 ## Uploading Source Maps
 
-todo: add source maps instructions
+To upload source maps, you can use the `sourceMapsUploadOptions` option inside the `sentry` options of your
+`nuxt.config.ts`:
+
+```javascript
+// nuxt.config.ts
+export default defineNuxtConfig({
+  modules: ['@sentry/nuxt/module'],
+  sentry: {
+    debug: true,
```

Comment:
hmm, no strong feelings, but it feels a bit confusing that this only turns on debug mode for the vite plugin, right? It seems easy to think that this also enables debug mode for sentry itself 🤔 would it make sense to name this in a different, more specific way, e.g. `viteDebug` or `debugVitePlugin`, something along these lines? Not sure how this works in other SDKs though, if we do the same there it's probably fine.

Reply:
I would keep a top-level debug option like this. Makes it very easy to ask users to enable debug mode to debug all issues related to build time stuff.

Reply:
I was thinking about the same actually before creating this PR. But this debug option is actually quite nice for the build-time debug. The debug flag in `init` only works during runtime. To make it more explicit, this could be renamed to `buildDebug`?

Reply:
I would keep it at `debug` tbh

Reply:
Fwiw (no strong opinions but from having dealt with this before): We have this pattern of multiple `debug` properties in different files in SvelteKit and Astro and so far people haven't complained about it. If we point out well in the JSDoc what kind of logging this specific option enables, I think going simply with `debug` is fine. 
Especially considering, this flag can be used everywhere within the nuxt module (so not just within the vite plugin I assume) I wouldn't give it a too specific name.